src/docfx/lib/json/JsonUtility.cs

```diff
@@ -105,17 +115,17 @@ public static string Serialize(object graph, bool indent = false)
         /// De-serialize a data string, which is not user input, to an object
         /// schema validation errors will be ignored, syntax errors and type mismatching will be thrown
         /// </summary>
-        public static T Deserialize<T>(string json, FilePath file) where T : class, new()
+        public static T Deserialize<T>(string json, FilePath file, bool checkAddionalContent = true) where T : class, new()
         {
             using var reader = new StringReader(json);
-            return Deserialize<T>(reader, file);
+            return Deserialize<T>(reader, file, checkAddionalContent);
```

Comment:
`string` does not need this switch, this can always be `true`
```suggestion
            return Deserialize<T>(reader, file, true);
```